### ISSUE TITLE
Two fields with the same name results in "<detail-undefined>"

### ISSUE THREAD

**Issue by JapSeyz** [eminiarts/nova-tabs#12]
Hey.

I have these two fields:

```php
Country::make('Country')->onlyOnForms(),
....
...
...
Text::make('Country')->exceptOnForms(),
```

Because the Form location and the detail location for the country is different - However it seems to clash and just default to Undefined in the JS.

**Comment by JapSeyz** [comment]
Experiencing the same then when having a single field with 3 modifiers:

```php
Boolean::make('Active', 'is_active')->hideWhenUpdating()->hideFromIndex()->hideFromDetail(),
```

This also results in the detail-undefined.

**Comment by bajramemini** [comment]
Hello @JapSeyz 

I am not able to reproduce this error.

When I do:

```php
Text::make('Name')->sortable()->rules('required', 'max:255')
    ->hideFromDetail()
,
Text::make('Name')->sortable()->rules('required', 'max:255')
    ->onlyOnDetail()
,
```

It works fine.

Could you show me your whole fields function?

**Comment by JapSeyz** [comment]
Hi @bajramemini I originally thought it was two fields with the same name, but it seems to happen to every field that I hide from the detail view, and then visiting the detail view.

The entire fields array looks like:

```php
(new TabsPanel(__('Car'), [
          Tabs::make()
              ->addTab(__('Details'), [
                Text::make('Name', 'name'),
                Country::make('Country')->onlyOnForms(),
                
                ])
              ->addTab(__('Address'), $this->addressFields())
        ]))->withToolbar(),
```

```php
/**
     * Get the address fields for the resource.
     *
     * @return \Illuminate\Http\Resources\MergeValue
     */
    protected function addressFields()
    {
        return [
            Text::make('Country')->exceptOnForms(),
            Place::make('Address', 'address_line_1')->hideFromIndex(),
            Text::make('Address Line 2')->hideFromIndex(),
            Text::make('City'),
            Text::make('State')->hideFromIndex(),
            Text::make('Postal Code')->hideFromIndex(),
        ];
    }
```

**Comment by bajramemini** [comment]
Ok thanks @JapSeyz 

Will have a look at it later and keep you updated.

**Comment by dkulyk** [comment]
Why need to add Tabs in TabsPanel?

**Comment by bajramemini** [comment]
@dkulyk 

You can now combine Fields and Relations in Tabs. [It's in the readme.](https://github.com/eminiarts/nova-tabs#combine-fields-and-relations-in-tabs).

That's why he is using Tabs in a TabsPanel.

**Comment by bajramemini** [comment]
Should be fixed in [1.0.0](https://github.com/eminiarts/nova-tabs/releases/tag/1.0.0)

Let me know if you have further issues.
